code/game/objects/structures/crates_lockers/crates/wooden.dm

```diff
@@ -15,4 +15,5 @@
 	new	/obj/item/reagent_containers/food/drinks/soda_cans/canned_laughter(src)
 	new /obj/item/pneumatic_cannon/pie(src)
 	new /obj/item/reagent_containers/food/snacks/pie/cream(src)
-	new /obj/item/storage/crayons(src)
\ No newline at end of file
+	new /obj/item/storage/crayons(src)
+	new /obj/item/bikehorn/rubber_pigeon(src) //yogs a single rubber pigeon
```

Comment:
🤔

Reply:
The rubber pigeon does not die so easily!